day_6/exercises/burrito.rb

```diff
@@ -1,19 +1,39 @@
-# Add the following methods to this burrito class and 
+# Add the following methods to this burrito class and
 # call the methods below the class:
 # 1. add_topping
 # 2. remove_topping
 # 3. change_protein
 
 class Burrito
-  attr_reader :protein, :base, :toppings
-  def initialize(protein, base, toppings)
+  attr_reader :protein, :base, :topping
+  def initialize(protein, base, topping)
     @protein  = protein
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(topping)
+    p "#{add_topping} has been added."
+  end
+
+  def remove_topping(topping)
+    p "#{topping} has been removed."
+  end
```

Comment:
What is the value of `toppings` after calling `dinner.add_topping("pico")`? Did it change?

How could you write the `add_topping` and `remove_topping` methods so the `toppings` array is changed?